getColorPair.py

```diff
@@ -0,0 +1,10 @@
+from main import *
+def get_color_from_pair_number(pair_number):
```

Comment:
Consistency: The counterpart of this file is called `PairNumberColor.py`, while this one starts with `get...`.
Try a consistent style.

Reply:
done!!!